tests/prepare/shell/test.sh

```diff
@@ -10,6 +10,10 @@ rlJournalStart
         rlRun "tmt run -rv plan -n custom" 0 "Prepare using a custom script"
     rlPhaseEnd
 
+    rlPhaseStartTest "Commandline Script"
+        rlRun "tmt run -arv plan -n custom prepare -h shell -s './prepare.sh'" 0 "Prepare using a custom script from cmdline"
```

Comment:
Would a second line `rlRun "tmt run -arv plan -n custom prepare -h shell -s './prepare.sh' -s './prepare.sh'" 0 "Prepare using custom scripts from cmdline"` make sense here? To check not just one but multiple scripts are accepted?

Reply:
As a 3rd test? Yes. I'll add something execute will check for.